src/final2/subtests/InvalidCommandTest.java

```diff
@@ -0,0 +1,915 @@
+package final2.subtests;
+
+import org.junit.Test;
+
+import test.Input;
+import test.runs.ErrorRun;
+import test.runs.NoOutputRun;
+import test.runs.Run;
+
+/**
+ * Tests the program for error messages with not well-formed or invalid commands.
+ * 
+ * @author Roman Langrehr
+ * @since 26.03.2015
+ * @version 1.0
+ *
+ */
+public class InvalidCommandTest extends LangtonSubtest {
+
+	private static final String[] ARGUMENT_COMMANDS = new String[] {
+			"create",
+			"field",
+			"position",
+			"direction",
+			"escape",
+			"move"
+	};
+
+	private static final String[] INVALID_ANT_NAMES = new String[] {
+			"ab",
+			"�",
+			"*"
+	};
+
+	private static final String[] INVALID_NUMBERS = new String[] {
+			"a",
+			"12t",
+			"three",
+			"12.5",
+			"$",
+			"9999999999999999999999999999999999999999999999999999999999999999"
+	};

```

Comment:
Yeah, I saw this issue just after creating this pull request, I'll add this test.